cranelift/filetests/filetests/runtests/f128-bitcast.clif

```diff
@@ -3,7 +3,7 @@ test run
 set enable_llvm_abi_extensions
 target x86_64
 target aarch64
-target aarch64 has_fp16
+target riscv64
```

Comment:
Is there any particular reason that we remove `aarch64 has_fp16` from the list of targets tested by this runtest and those below? Can we keep that while adding `riscv64` below it?

Reply:
Ah, right, of course -- saw the seemingly unrelated change but that cleanup makes sense, thanks.